lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3741,8 +4259,12 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
     setAdjointBuffer(dsi->getOperand(),
         ValueWithCleanup(adjBuf, makeCleanup(adjBuf,
             [](SILBuilder &b, SILLocation l, SILValue v) {
-              b.createReleaseValueAddr(l, v, b.getDefaultAtomicity());
+              if (v->getType().isLoadable(b.getModule()))
+                b.createReleaseValueAddr(l, v, b.getDefaultAtomicity());
+              else
+                b.createDestroyAddr(l, v);
             })));
+    // localAllocations.push_back(adjBuf);
```

Comment:
Remove.

Reply:
Done in https://github.com/apple/swift/pull/22544/commits/c1ff15405abb1df3ec85d58a0b40df718376a90f.